src/syngen/ml/data_loaders/data_loaders.py

```diff
@@ -150,6 +150,8 @@ def _load_data(path, **kwargs) -> Tuple[pd.DataFrame, Dict]:
             if len(sep) > 1:
                 params["sep"] = ","
             params["skiprows"] = None
+            if params.get("na_values", []):
+                params["na_rep"] = params["na_values"][0]
```

Comment:
Is this [0] indexing safe when .get returns empty list?

Reply:
Yes, this check whether the list is empty or not in the line 153